src/ast/passes/codegen_llvm.cpp

```diff
@@ -462,8 +462,13 @@ void CodegenLLVM::visit(Builtin &builtin)
     expr_ = b_.CreateLoad(b_.GetType(builtin.type), dst);
     b_.CreateLifetimeEnd(dst);
   } else if (builtin.ident == "probe") {
-    builtin.probe_id = get_probe_id();
-    expr_ = b_.getInt64(builtin.probe_id);
+    auto probe_str = probefull_;
+    probe_str.resize(builtin.type.GetSize() - 1);
```

Comment:
Why `-1`?

Reply:
Because we added one when we created the type (for the null terminator I'm guessing - we do something similar for the string ast)
```
CreateString(str_size + 1);
```